doc/nrf/templates/application_README.rst

```diff
@@ -9,7 +9,7 @@ Template: Application
 
 .. note::
    * Provide a concise name to the application, which corresponds to the folder name.
-     If the application targets a specific device, add it in the title before the application name (for example, "nRF9160:").
+     If the application targets a specific device, add it in the title before the application name (for example, "nRF5340:").
```

Comment:
Maybe add an 'or' to give an example for the nrf91 series, or is it no longer possible to do this?

Reply:
Applications that support RF91 Series devices won't have the prefix as they support more than one device. For example, Asset Tracker v2 application. Hence, we could keep this as is.